ios/accessibility/accessibility.go

```diff
@@ -14,6 +14,5 @@ func New(device ios.DeviceEntry) (ControlInterface, error) {
 		return ControlInterface{}, err
 	}
 	control := ControlInterface{conn.GlobalChannel()}
-	err = control.init()
```

Comment:
does this break the call to `deviceResetToDefaultAccessibilitySettings` if this happens here?

Reply:
yes, I noticed that the `init()` panics here all the time https://github.com/danielpaulus/go-ios/pull/546/files#diff-d0706263521565437e277a55d7e4feb44b794334cb4b3bb64de0afbead4a2656R69

Reply:
in any case I am introducing breaking changes here

Reply:
> in any case I am introducing breaking changes here

we could avoid that by adding a `NewWithoutEventChangeListeners` instead and keep the `New` as it is